spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/config/AdminConfiguration.java

```diff
@@ -158,9 +163,13 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 	}
 
 	@Bean
-	@ConditionalOnMissingBean
+	@ConditionalOnMissingBean(TapOnChannelRecoveryStrategy.class)
```

Comment:
I think it is always a safe thing to add `@ConditionalOnMissingBean` at the `starter` project to avoid any conflicts in general and adding this specific type here would make it better for the project that uses this `starter` project.